### ISSUE TITLE
Don't output markup when captcha is disabled

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#16887]
### Steps to reproduce the issue
On frontend, click `Forgot your username?` and `Forgot your password?`.
There is a large gap between Email Address and Submit button.
View page source to see markup for captcha even though captcha is disabled.

### Expected result
No markup for captcha.

### Actual result
```
							<div class="control-group">
					<div class="control-label">
											</div>
					<div class="controls">
											</div>
				</div>
```

### System information (as much as possible)
3.7.3-nightly build
protostar template

**Comment by brianteeman** [comment]
I can confirm this behaviour but it makes no sense to me why it happens

**Comment by zero-24** [comment]
Comes from here: https://github.com/joomla/joomla-cms/blob/staging/components/com_users/views/remind/tmpl/default.php and https://github.com/joomla/joomla-cms/blob/staging/components/com_users/views/reset/tmpl/default.php

It looks like we need to add a check there if nothin is returned or similiar.

**Comment by zero-24** [comment]
Or better qould be that the field is not returned using that method but this needs to be debugged.

**Comment by brianteeman** [comment]
yes I saw that BUT doesnt that if statement get the code for the remind/reset input field so I dont see why it is being run again for the captcha

**Comment by zero-24** [comment]
https://github.com/joomla/joomla-cms/blob/staging/components/com_users/models/forms/remind.xml

The view has the captcha field so it looks like it is also returned but as empty field. But i hope i can find some time to debug that and see why the method returned empty fields.

**Comment by brianteeman** [comment]
AH now I understand the why  :)

**Comment by zero-24** [comment]
This should work: https://github.com/joomla/joomla-cms/pull/16933 but I'm not sure if that is the correct way to fix the issue.

**Comment by franz-wohlkoenig** [comment]
closed as having PR #16933<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16887">issues.joomla.org/tracker/joomla-cms/16887</a>.</sub>
